kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDataPanel.js

```diff
@@ -191,11 +191,12 @@ define([
                     this.trigger('hideGalleryPanelOverlay.Narrative');
                     this.trigger('toggleSidePanelOverlay.Narrative', this.$overlayPanel);
 
-                    // NOTE - this will be missed and a widget will remain active if the panel is closed by means other than clicking this button.
+                    // NOTE - this will be missed and a widget will remain active if the panel 
+                    // is closed by means other than clicking this button.
                     // This should be re-visited at some point.
                     this.deactivateLastRenderedPanel();
 
-                    //once we've clicked it 10 times, meaning we've open and shut the browser 5x, we reveal its TRUE NAME.
+                    // once we've clicked it 10 times, meaning we've open and shut the browser 5x, we reveal its TRUE NAME.
```

Comment:
Exciting KBase easter egg?!?!

Reply:
Yeah, a former KBaser snuck that in an age ago. It didn't hurt anything and was kinda cute, so we left it. :)